{
    "code": 200,
    "msg": "success",
    "data": [
        {
            "id": 1,
            "level": "重要",
            "service": "主机B->hadoop-slave1-NodeManager",
            "kpi": "主机B->hadoop-slave1:cpu",
            "daily": "Caused by: java.net.NoRouteToHostException",
            "config": "configtest",
            "path": "ResourceManager(主机A,hadoop-master)——>NodeManager(主机B,hadoop-slave1)",
            "time": "2020-12-27 10:01:46"
        },
        {
            "id": 2,
            "level": "重要",
            "service": "主机B->hadoop-slave1-NodeManager",
            "kpi": "主机B->hadoop-slave1:cpu",
            "daily": "Caused by: java.net.NoRouteToHostException",
            "config": "configtest",
            "path": "",
            "time": "2020-12-27 10:01:46"
        },
        {
            "id": 3,
            "level": "重要",
            "service": "主机B->hadoop-slave1-NodeManager",
            "kpi": "主机B->hadoop-slave1:cpu",
            "daily": "Caused by: java.net.NoRouteToHostException",
            "config": "configtest",
            "path": "",
            "time": "2020-12-27 10:01:46"
        },
        {
            "id": 4,
            "level": "重要",
            "service": "主机B->hadoop-slave1-NodeManager",
            "kpi": "主机B->hadoop-slave1:cpu",
            "daily": "Caused by: java.net.NoRouteToHostException",
            "config": "configtest",
            "path": "",
            "time": "2020-12-27 10:01:46"
        },
        {
            "id": 5,
            "level": "重要",
            "service": "主机B->hadoop-slave1-NodeManager",
            "kpi": "主机B->hadoop-slave1:cpu",
            "daily": "Caused by: java.net.NoRouteToHostException",
            "config": "configtest",
            "path": "",
            "time": "2020-12-27 10:01:46"
        },
        {
            "id": 6,
            "level": "重要",
            "service": "主机B->hadoop-slave1-NodeManager",
            "kpi": "主机B->hadoop-slave1:cpu",
            "daily": "Caused by: java.net.NoRouteToHostException",
            "config": "configtest",
            "path": "",
            "time": "2020-12-27 10:01:46"
        },
        {
            "id": 7,
            "level": "重要",
            "service": "主机B->hadoop-slave1-NodeManager",
            "kpi": "主机B->hadoop-slave1:cpu",
            "daily": "Caused by: java.net.NoRouteToHostException",
            "config": "configtest",
            "path": "",
            "time": "2020-12-27 10:01:46"
        },
        {
            "id": 8,
            "level": "重要",
            "service": "主机B->hadoop-slave1-NodeManager",
            "kpi": "主机B->hadoop-slave1:cpu",
            "daily": "Caused by: java.net.NoRouteToHostException",
            "config": "configtest",
            "path": "",
            "time": "2020-12-27 10:01:46"
        }
    ]
}